policyengine_us/parameters/gov/states/ca/tax/income/alternative_minimum_tax/exemption/amti/threshold/lower.yaml

```diff
@@ -0,0 +1,30 @@
+description: California provides a flat alternative minimum tax exemption amount for filers with alternative minimum taxable income below this amount, based on filing status.
+metadata:
+  label: California alternative minimum tax exemption lower AMTI threshold 
+  period: year
+  unit: currency-USD
+  breakdown:
+  - filing_status
+  reference:
+  - title: 2021 Schedule P (540) Alternative Minimum Tax and Credit Limitations - Residents
+    href: https://www.ftb.ca.gov/forms/2021/2021-540-p.pdf#page=1
+  - title: 2022 Schedule P (540) Alternative Minimum Tax and Credit Limitations - Residents
+    href: https://www.ftb.ca.gov/forms/2022/2022-540-p.pdf#page=1
+  - title: Cal. Rev. & Tax. Code § 17062 
```

Comment:
define subsection and leave a comment saying what year the legal code defines it for (same for other parameters)

Reply:
the following parameter:
policyengine_us/parameters/gov/states/ca/tax/income/alternative_minimum_tax/exemption/amti/threshold/upper.yaml
is not specified in the [legal code ](https://casetext.com/statute/california-codes/california-revenue-and-taxation-code/division-2-other-taxes/part-10-personal-income-tax/chapter-21-alternative-minimum-tax/section-17062-alternative-minimum-tax) but also does not match any of the existing parameters in the federal AMT structure, not sure if I am completely missing something here